software/control/_def.py

```diff
@@ -875,3 +875,11 @@ def load_formats():
 # saving path
 if not (DEFAULT_SAVING_PATH.startswith(str(Path.home()))):
     DEFAULT_SAVING_PATH = str(Path.home()) + "/" + DEFAULT_SAVING_PATH.strip("/")
+
+# fluidics
+RUN_FLUIDICS = False
+PORT_LIST = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24]
+FLUIDICS_CONFIG_PATH = "./merfish_config/MERFISH_config.json"
+FLUIDICS_SEQUENCE_PATH = "./merfish_config/merfish-imaging.csv"
+BEFORE_IMAGING_SEQUENCES = [0, 3]
+AFTER_IMAGING_SEQUENCES = [4, 6]
```

Comment:
Instead of using `RUN_FLUIDICS` we should have a dedicated widget running fluidics + imaging. The fluidics sequence should be loaded there.